projects/cups/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/apple/cups"
+primary_contact: "skau@chromium.org"
+sanitizers:
+  - address
+  - leak
```

Comment:
I don't think we support "leak" as a separate tool (leaks will be detected by AddressSanitizer)